### ISSUE TITLE
Details of the test set and training set

### ISSUE THREAD

**Issue by anothersin** [sword98/CKD-TransBTS#3]
Thank you very much for your contribution to medical segmentation! To make it easier to follow your work, could you provide a list of specific training, validation and test set documents?

**Comment by sword98** [comment]
Thanks, we have uploaded the list of training, validation and test set to the `dataset/` folder. You can download the BraTS2021 from the [official website](https://www.synapse.org/#!Synapse:syn25829067/wiki/).

**Comment by CodeHarcourt** [comment]
But in the official website, BraTS 2021 had update to BraTS 2023, where can i get the specific training, validation and test set documents?

**Comment by faizan1234567** [comment]
@CodeHarcourt same. @Milkomeda98 could you please let us know how can we use this code with BraTS23 dataset since BraTS21 dataset is updated to make BraTS23. Therefore, BraTS21 dataset is not available on the website you pointed.

**Comment by faizan1234567** [comment]
@CodeHarcourt have you found any solution? If yes, kindly share it with us.
